src/pages/home/report/ReportActionContextMenu/ContextActions.js

```diff
@@ -0,0 +1,35 @@
+import {
+    ClipboardIcon, LinkCopyIcon, MailIcon, PencilIcon, TrashIcon,
+} from '../../../../components/Expensicons';
+
+export default [
+    // Copy to clipboard
+    {
+        text: 'Copy to Clipboard',
+        icon: ClipboardIcon,
+    },
+
+    // Copy chat link
+    {
+        text: 'Copy Link',
+        icon: LinkCopyIcon,
+    },
+
+    // Mark as Unread
+    {
+        text: 'Mark as Unread',
+        icon: MailIcon,
+    },
+
+    // Edit Comment
+    {
+        text: 'Edit Comment',
+        icon: PencilIcon,
+    },
+
+    // Delete Comment
+    {
+        text: 'Delete Comment',
+        icon: TrashIcon,
+    },
+];
```

Comment:
Same with this. We might as well put this in the same file as where it will be used.